llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2831,6 +2840,18 @@ class VPRegionBlock : public VPBlockBase {
   /// VPRegionBlock.
   VPBlockBase *Exiting;
 
+  /// Hold the Early Exiting block of the SEME region, if one exists.
+  VPBlockBase *EarlyExit;
```

Comment:
IIUC, for the SEME region, the latch exit VPBlock `middle.block` is placed as the successor of the region block, and the other exit VPBlocks other than the latch exit VPBlock are included in the region block, e.g. `vector.early.exit` pointed by `EarlyExit`.
And the early exiting block would be the predecessor of `EarlyExit`. 
Therefore the comment should be updated to `/// Hold the Early Exit block of the SEME region, if one exists`.